enterprise/signals.py

```diff
@@ -47,4 +47,6 @@ def handle_user_post_save(sender, **kwargs):  # pylint: disable=unused-argument
         enterprise_customer=pending_link_record.enterprise_customer,
         user_id=user_instance.id
     )
+    for enrollment in pending_link_record.pendingenrollment_set.all():
```

Comment:
Please add a code comment describing what is happening in this new addition to the workflow.  Also please update the docstring to mention this new work as well.